### DISCUSSION TITLE
SMTP None Authentication

### DISCUSSION THREAD

**Discussion by Moubasher82** [uvdesk/community-skeleton#733] [General]
Dear All,

I am trying to setup my SwiftMailer but I have only tls and ssl option, I am using local email server without authentication, how can i fix this issue please.

I have the latest uvdesk installed.

Thanks a lot

**Comment by Moubasher82** [comment]
dear all, any help please?

**Comment by juggernautsei** [comment]
How to send a test email? I could not find a way to send a test email from the system. There is nothing labeled email test that I could find.
